lib/shelley/test/integration/Main.hs

```diff
@@ -133,21 +135,22 @@ main = withUtf8Encoding $ withLogging Nothing Info $ \(_, tr) -> do
             describe "Mnemonics CLI tests" $ parallel (MnemonicsCLI.spec @t)
             describe "Miscellaneous CLI tests" $ parallel (MiscellaneousCLI.spec @t)
             describe "Key CLI tests" $ parallel (KeyCLI.spec @t)
-        specWithServer tr $ do
-            describe "API Specifications" $ do
-                Addresses.spec @n
-                Transactions.spec @n
-                Wallets.spec @n
-                HWWallets.spec @n
-                Network.spec
-                Network_.spec
-            describe "CLI Specifications" $ do
-                AddressesCLI.spec @n
-                TransactionsCLI.spec @n
-                WalletsCLI.spec @n
-                HWWalletsCLI.spec @n
-                PortCLI.spec @t
-                NetworkCLI.spec @t
+        describe "API Specifications" $ specWithServer tr $ do
```

Comment:
Hey! No! 

There's a wrong merge-conflict resolution here I think ^.^ .. I moved the `specWithServer` above to be around ALL, otherwise, this actually restarts a cluster between both steps. Not only does it takes time, but it also doubles the number of genesis credentials we need for starting stake pools. The `Migrations.spec @n` can be added all the same.

Reply:
oh ok, whatever it was... it was done automatically (I think) :sweat_smile: I'll fix.